File: 3228_Maximum_Number_of_Operations_to_Move_Ones_to_the_End/solution.py
class Solution:
    def maxOperations(self, s: str) -> int:
        n = len(s)
        ones = 0
        operations = 0

        for i in range(n):
            if s[i] == "1":
                ones += 1
            elif s[i] == "0" and i > 0 and s[i - 1] == "1":
                operations += ones

        return operations

File: 3228_Maximum_Number_of_Operations_to_Move_Ones_to_the_End/solution.c
int maxOperations(char* s) {
    int n = strlen(s);
    int ones = 0;
    int operations = 0;

    for (int i = 0; i < n; i++) {
        if (s[i] == '1') {
            ones++;
        } else if (s[i] == '0' && i > 0 && s[i - 1] == '1') {
            operations += ones;
        }
    }

    return operations;
}